gapic-generator/lib/google/gapic/generators/base_generator.rb

```diff
@@ -69,16 +69,33 @@ def controller
           @controller ||= Class.new(ActionController::Base).new
         end
 
-        def gen template, filename, **args
+        def generate_file template, filename, format: true, **args
```

Comment:
I really like the addition of this `format` option.

Reply:
This change adds the private method `format_file` that can be overridden if users don't want to use Rubocop to perform the formatting.